test.py

```diff
@@ -14,24 +14,37 @@
 
```

Comment:
As silly as it may sound, since this file was changed to be executable (755) instead of 644 like it was before, `nosetests` actually skips over it, so none of the unit tests are actually running for this PR.